performance_management/views/api.py

```diff
@@ -161,11 +177,23 @@ def get(self, request):
                 quarter = 0
             quarter += 1
 
-            activity_count = Activity.objects.filter(
+            activities = Activity.objects.filter(
                 Q(activity_date__date__gte=datetime.datetime(self.start_year, quarter_months[quarter][0], 1)) &
-                Q(activity_date__date__lte=datetime.datetime(self.start_year, quarter_months[quarter][2],
-                                                             1))).distinct().count()
-            summary.append({'quarter': quarter, 'activity_count': activity_count, 'year': self.start_year})
+                Q(activity_date__date__lte=datetime.datetime(self.start_year, quarter_months[quarter][2],1))
+            ).distinct()
+
+            activities_count = activities.count()
+                
+            completed = activities.filter(activity_status__status="Completed").values('activity_status').count()
+            
+            ongoing = activities.filter(activity_status__status="Ongoing").values('activity_status').count()
+
+            not_done = activities.filter(activity_status__status="Not Done").values('activity_status').count()
+
+            # Statuses not equal to completed, ongoing and not_done
+            rest = activities.filter(~Q(activity_status__status="Not Done") | ~Q(activity_status__status="Completed") | ~Q(activity_status__status="Completed")).values('activity_status').count()
```

Comment:
I don't think **rest** is an option. Activity must fall under one of those states; completed, not done and ongoing

Reply:
@codephillip actually i wasnt supposed to push this. This one is not working. I was trying to see if i could capture all else in rest and see if the total tallies